users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -829,6 +829,25 @@ def run_scf_specaug():
                 lr_args=lr_args,
                 report_args={"batch_size": "2x5k", "enable_sorting": False, "max_feature": 15},
             ),
+            "variance_based_specaug": dict(
+                returnn_args={
+                    **base_returnn_args,
+                    "specaug_config": {
+                        "enable_sorting": False,
+                        "mask_based_on_variance": True,
+                        "variance_factor": 0.5,
+                        "total_max_number_masks": 50,
```

Comment:
This is less than usual, right? You told me offline that the maximum masking in frequency is 9.4% of the channels on average. This here would be less.

Reply:
Also, you need to change the variable name.